scala3/getting-started.md

```diff
@@ -69,15 +69,15 @@ sbt compiles, runs, and tests your Scala code.
 To create a new Scala project with sbt:
 
 1. `cd` to an empty folder.
-1. Run this command `sbt new scala/hello-world.g8`.
+1. Run this command `sbt new scala/scala3.g8`.
 This pulls the ['hello-world' template][template-url] from GitHub.
 It also creates a _target_ folder, which you can ignore.
-1. When prompted, name the application `hello-world`.
-   This will create a project called "hello-world".
+1. When prompted, name the application `scala-3-project-template`.
+   This will create a project called "scala-3-project-template".
```

Comment:
I don't think it makes sense to name the generated project template. scala3.g8 is the template.

Reply:
indeed, but I couldn't find any better alternative for **Hello world** in Scala 3. Even it is a template it is still Scala 3 while the original was Scala 2. I do believe it is better for end user (who wants to try Scala 3 project)

Are you aware of any Scala 3 Hello world project?

Reply:
I was thinking and removed the word `template` - does it a help a bit?

Reply:
Thanks for catching this one @xhudik. It looks good to me. Unless @Philippus objects, I would go ahead and merge.